julia.spdx.json

```diff
@@ -0,0 +1,124 @@
+{
+    "spdxVersion": "SPDX-2.2",
+    "dataLicense": "CC0-1.0",
+    "SPDXID": "SPDXRef-DOCUMENT",
+    "name": "julia-spdx",
+    "documentNamespace": "https://julialang.org/spdxdocs/julia-spdx-26ce6dfe-7843-4342-9970-dda3d15f01e8",
+    "creationInfo": {
+        "creators": [
+            "Organization: julialang.org ()",
+            "Person: Simon Avery ()"
```

Comment:
What is this section referring to? Who is Simon Avery?

Reply:
It's purpose to to identify who/what (organization, person, and/or automated tool) created the SPDX document.  The Julia project gets first credit since this document exists in the repository.  I chose "julialang.org" as the identifier but this can be changed if you have another preference.  The parenthesis is for an optional email address. I didn't what to put there for the Julia project so I left it blank.

I'm Simon Avery.  I figured why not when I was creating this. But I'm fine with deleting my name if that is preferred.

Reply:
No, that's fine. It wasn't clear to me that you were Simon Avery (it's not in your GitHub profile). Carry on. It might be good to have email addresses, but I guess the question becomes which ones. If you put yours there, then you'll get contacted about this kind of stuff which you may not want.

Reply:
Exactly. Does the Julia project have a public facing email address you’d like to put in?

Reply:
Hmm. Let's just leave it blank for now.